.vscode/settings.json

```
@@ -5,5 +5,6 @@
 	"editor.defaultFormatter": "esbenp.prettier-vscode",
 	"editor.formatOnSave": true,
 	"eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
+	"eslint.probe": ["javascript", "javascriptreact", "typescript", "typescriptreact", "html", "vue", "markdown", "jsonc", "json"],
```

This template doesn't have tooling support for `"html"` or `"vue"` files. I think we can keep them out. There are a ton of random other languages that folks could add tooling support for on their own (CSS, Less, etc.) that this template doesn't support. What do you think?

```suggestion
	"eslint.probe": ["javascript", "javascriptreact", "typescript", "typescriptreact", "markdown", "jsonc", "json"],
```

---

Please have a look at it now.